### PR TITLE
Combat System & Movement System 

### PR DESCRIPTION
**Pull request by johnw08**
Combat System:
Weapon Equipping: "E"
Weapon Switching: "Tab"; Two weapons available
Attack one: "J"
Attack Two: "K"
Special Attack: "U"

Movement System:
Run: Hold "Shift"
Jump: "Space"
Sprint Jump: "WASD" + "Space"

Combat system & Movement System Integration

### CODE REVIEW

File: Assets/Prefab/BloodSprayFX.prefab
Comment (AlstonXiao) [inline Assets/Prefab/BloodSprayFX.prefab:1]: What this is??

File: Assets/Script/Enemy/EnemyAiTutorial.cs
Comment (AlstonXiao) [inline Assets/Script/Enemy/EnemyAiTutorial.cs:1]: Why there is a enemy AI and enemy script?

File: ProjectSettings/TagManager.asset [hunk 1]
```diff
@@ -3,16 +3,17 @@
...
   - Default
   - TransparentFX
   - Ignore Raycast
   - Environment
   - Water
   - UI
-  - 
-  - 
+  - Weapon
+  - Enemy
```
Comment (AlstonXiao) [inline ProjectSettings/TagManager.asset:16]: Not sure if this is the best way to get layers. Maybe good for now.

**Review by AlstonXiao — COMMENTED** [review]
I think it LGTM but we need some tutorial of all the player script by Saturday.

File: Assets/Prefab/BloodSprayFX.prefab
Comment (johnw08) [reply Assets/Prefab/BloodSprayFX.prefab:1]: > What this is??

This is the prefab for the blood effect which we are currently not using. We can delete it.

File: Assets/Script/Enemy/EnemyAiTutorial.cs
Comment (johnw08) [reply Assets/Script/Enemy/EnemyAiTutorial.cs:1]: > Why there is a enemy AI and enemy script?

This is from the tutorial which I intend to follow later.

**Comment by johnw08** [comment]
> I think it LGTM but we need some tutorial of all the player script by Saturday.

We can talk about the script during the meeting.

**Review by jialuogao — APPROVED** [review]

**Pull request merged**